### DISCUSSION TITLE
Project (beta): Impossible to go to closed projects

### DISCUSSION THREAD

**Discussion by g3offrey** [community/community#6064] [Projects and Issues]
Hi,

First of all thank you for your awesome work on this awesome feature.
Day after day I love GitHub a bit more ❤️ 

I just encountered a bug. As I am totally new to this beta, I don't know if this is the right place to show it.
I searched about it on **Discussions** but didn't find any information about it.

When I try to go to **Projects (Beta) -> Closed projects** it redirects me to **Projects -> Closed projects** where I can't find the closed project I did.

Here is a gif that show this issue : 
![Enregistrement de l’écran 2021-10-07 à 12 39 45](https://user-images.githubusercontent.com/11151445/136369567-f98270c1-0214-4c45-a407-6769d4d380ef.gif)

I investigated on my side and found the issue.
In fact this link goes to : `https://github.com/g3offrey?tab=projects&q=is:closed`
instead of `https://github.com/g3offrey?tab=projects&q=is:closed&type=beta`

It seems the same happen when you try to go to open project.

Hope this feedback will help you 😄

**Comment by julianxhokaxhiu** [comment]
Happens here as well. Please consider fixing the link.
